src/t-i18n.ts

```diff
@@ -62,7 +62,7 @@ const makeIntlFormatters = (locale: () => string): IntlFormatters => {
 			args[0] = args[0] || "en-US";
 			args[1] = args[1] || {};
 			args[1].timeZone = args[1].timeZone || "America/Toronto";
-			return delegate.apply(this, args);
+			return delegate.apply(this, args as [string, Intl.DateTimeFormatOptions]);
```

Comment:
It'd be nice to avoid type coercion here if possible. Instead of slicing the arguments above, we could just validate each of them. For example, instead of lines 61 and 62, we could have `const lang = typeof arguments[0] === "string" ? arguments[0] : "en-US";`, and so on.